src/components/Sidebar/SearchResults.js

```diff
@@ -31,8 +36,18 @@ function SearchResults({
       </div>
     );
   }
+  if (searchResults.length === 0 && currentSearchInput.length >= 3) {
+    return (
+      <div className={classes.inputHint}>
+        No results
+      </div>
+    );
+  }
   return (
     <div className={classes.searchResults}>
+      {currentSearchInput.length > 0 && currentSearchInput.length <= 2
```

Comment:
Can you think of any cases when the > 0 check would matter? I can't think of any... I think it would be fine just having the <= 2 check.

Reply:
Yes, because the currentSearchInput initializes as an empty string, so without that check, it would prompt the user to keep typing as soon as the app loads